.bazelrc

```diff
@@ -5,3 +5,22 @@ build --crosstool_top=@nixpkgs_config_cc//:toolchain
 # resolved before `--incompatible_enable_cc_toolchain_resolution` can be
 # recommended for `nixpkgs_cc_configure_hermetic`.
 # build --incompatible_enable_cc_toolchain_resolution
+
+# CI Configuration
+# ----------------
+common:ci --color=no
+build:ci --verbose_failures
+test:ci --test_output=errors
+
+# Use a remote cache during CI
+build:ci --bes_results_url=https://app.buildbuddy.io/invocation/
+build:ci --bes_backend=grpcs://cloud.buildbuddy.io
```

Comment:
For one second I thought this was remote build. Would have been quite a surprise to see it working